pom.xml

```diff
@@ -71,7 +71,7 @@
     <maven.resources.plugin.version>2.6</maven.resources.plugin.version>
     <maven.dependency.plugin.version>2.7</maven.dependency.plugin.version>
     <maven.assembly.plugin.version>2.4</maven.assembly.plugin.version>
-    <maven.javadoc.plugin.version>3.0.0-M1</maven.javadoc.plugin.version>
+    <maven.javadoc.plugin.version>2.10.1</maven.javadoc.plugin.version>
```

Comment:
@johnou need to take another shot at this. The last plugin version change broken JavaDoc generation and I haven't been able to get it working yet.